operator/api/redpanda/v1alpha2/redpanda_clusterspec_types.go

```diff
@@ -149,6 +149,10 @@ type Enablable struct {
 type ConfigWatcher struct {
 	// Specifies whether the sidecar is enabled.
 	Enabled *bool `json:"enabled,omitempty"`
+	// Skip cluster configuration is a flag changes the behavior of sidecar to not synchronize
```

Comment:
This field shouldn't be present in the CRD as we don't want to allow users to change this configuration. An exception will have to be added to the property test when the v5 chart version is bumped in this branch.